tool/kokoro/setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Initialize everything required by the plugin tool.
+setup() {
+  # Fail on any error.
+  set -e
+
+  # Display commands being run. Only do this while debugging and be careful
+  # that no confidential information is displayed.
+  # set -x
+
+  git clone --depth 1 https://github.com/flutter/flutter.git ../flutter
```

Comment:
This shallow clone is faster, but this checkout of flutter won't have accurate access to things like its version info (which is stored in git metadata). I assume this won't be a problem for our use of Flutter here, but FYI if we run into issues down the road.